ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -14,10 +14,13 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using ReactNative.Chakra;
 using static System.FormattableString;
 
 namespace ReactNative.Bridge
 {
+    public delegate void OnTransitionToBridgeHandler(bool busy);
```

Comment:
In general, we try to avoid multiple classes (even delegates) in one file.

Reply:
#fixed